prody/sequence/analysis.py

```diff
@@ -712,7 +712,7 @@ def alignSequencesByChain(PDBs, **kwargs):
     :type join_char: str
     """
     
-    if not isscalar(PDBs):
+    if not isinstance(PDBs, list) and not isinstance(PDBs, ndarray) and not isinstance(PDBs, tuple):
```

Comment:
Is isscalar not working?
You can do this which is equivalent but shorter:
isinstance(PDBs, (list, ndarray, tuple))

Reply:
Ok I’ll do that then. I tried it with a list instead of a tuple and that didn’t work

Reply:
isscalar didn't work on a list? It should though. isscalar should be more comprehensive than manually checking enumerable types using isinstance. Do you have an example code that didn't work, and I will look into it. Because this is not the only place I use isscalar for type checking and if it is not working then I will replace them all.

Reply:
Iterables are the opposite of scalars. Scalars are single entry things like int and str. If we use `if isscalar` instead of `if not isscalar`

Reply:
You are absolutely correct. It is a mistake to use `not`. `isscalar` should work just fine.